MobileCenter/MobileCenter/Internals/Channel/MSLogManagerDefault.m

```diff
@@ -82,7 +86,7 @@ - (void)enumerateDelegatesForSelector:(SEL)selector withBlock:(void (^)(id<MSLog
 
 #pragma mark - Process items
 
-- (void)processLog:(id<MSLog>)log withPriority:(MSPriority)priority {
+- (void)processLog:(id<MSLog>)log withPriority:(MSPriority)priority andGroupID:(NSString *)groupID {
```

Comment:
I wonder if you can get rid of a priority parameter from all (or most of) methods.